config/packages/assets.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+    assets:
+        json_manifest_path: '%kernel.project_dir%/public/build/manifest.json'
```

Comment:
@TomasVotruba i guess this config is autogenerated.

`An exception has been thrown during the rendering of a template ("Asset manifest file "/var/www/pehapkari.cz/public/build/manifest.json" does not exist.").`

It is related, because it points to non existing path

Reply:
Yes. The file is generated by `npm run build`
It has to be part of the build (not sure where exactly)

Reply:
Ok, if it is part of the build process, Dockerfile is the right place for it ;-)

Reply:
I can do it for you.

Reply:
> Ok, if it is part of the build process, Dockerfile is the right place for it ;-)

I see.

Something like?

```bash
# Dockerfile
# composer install...

RUN npm install
RUN npm run build
```

> I can do it for you.

:love